definitions/README.md

```diff
@@ -2,3 +2,84 @@
 
 This directory contains definitions for specific metrics, data sources, dimensions or segments for the platform they target.
 These definitions can be referenced in project specific configuration files without having to be redefined.
+
+Metric definitions can be validated using the following Python script in Google Colab:
+
+First, ensure that `mozanalysis` is available:
+
+```shell
+!pip install mozanalysis -qq
+```
+
+Second, setup the environment:
+
+```python3
+import pandas as pd
+
+from google.cloud import bigquery
+from google.colab import auth
+from mozanalysis.config import ConfigLoader
+from textwrap import dedent
+
+auth.authenticate_user()
+
+def fetch_metric_hub_definition(start_date, end_date, metric_slug, app_name, bq_project):
+    start_date = pd.to_datetime(start_date).date()
+    end_date = pd.to_datetime(end_date).date()
+
+    # Set useful attributes based on the Metric Hub definition
+    metric = ConfigLoader.get_metric(metric_slug=metric_slug, app_name=app_name)
+    submission_date_column = metric.data_source.submission_date_column
+
+    # Modify the metric source table string so that it formats nicely in the query.
+    from_expression = metric.data_source._from_expr.replace("\n", "\n" + " " * 15)
+
+    query = dedent(
+        f"""
+        SELECT {submission_date_column},
+               {metric.select_expr} AS value
+          FROM {from_expression}
+         WHERE {submission_date_column} BETWEEN '{start_date}' AND '{end_date}'
+         GROUP BY {submission_date_column}
```

Comment:
I'd suggesting adding `         ORDER BY {submission_date_column}` under here so that the results are always in chronological order.